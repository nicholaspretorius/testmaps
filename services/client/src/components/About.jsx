import React from "react";

const About = () => {
  return (
    <div>
      <h3>About Us</h3>
      <div>
        <p className="content">About us content goes here.</p>
      </div>
    </div>
  );
};

export default About;
