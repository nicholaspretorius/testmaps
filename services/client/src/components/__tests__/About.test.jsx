import React from "react";
import { render, cleanup } from "@testing-library/react";

import About from "./../About";

afterEach(cleanup);

it("renders properly", () => {
  const { getByText } = render(<About />);
  expect(getByText("About us content goes here.")).toHaveClass("content");
});

it("renders", () => {
  const { asFragment } = render(<About />);
  expect(asFragment()).toMatchSnapshot();
});
