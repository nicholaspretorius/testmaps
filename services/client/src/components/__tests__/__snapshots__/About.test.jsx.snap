// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div>
    <h3>
      About.
    </h3>
    <div>
      <p
        class="content"
      >
        Wakemaps is a wakepark listing directory.
      </p>
    </div>
  </div>
</DocumentFragment>
`;
